packages/errors/src/context.ts

```diff
@@ -399,6 +400,9 @@ export type SolanaErrorContext = DefaultUnspecifiedErrorContextToUndefined<
             index: number;
             instructionErrorContext?: unknown;
         };
+        [SOLANA_ERROR__INSTRUCTION_PLANS__FAILED_TO_EXECUTE_TRANSACTION_PLAN]: {
+            transactionPlanResult: unknown;
+        };
```

Comment:
@steveluscher This problem is back. However, this time we cannot omit this attribute or reduce it to scalar values. Returning the plan is crucial for the app to land on its feet since it can then grab the part of the plan that did not succeed at try again. Perhaps here we should try the non-writable object property trick?

Reply:
This required changes to the `SolanaError` constructor so I did the work on [a separate PR](https://github.com/anza-xyz/kit/pull/674).